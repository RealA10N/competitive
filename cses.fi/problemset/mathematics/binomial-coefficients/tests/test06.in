4
956335 24750
403107 56375
789618 592722
274758 54695
