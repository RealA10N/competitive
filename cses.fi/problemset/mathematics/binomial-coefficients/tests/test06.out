810998679
976885351
30288998
3319358
